docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -5,4 +5,42 @@ weight: 50
 featureId: build.buildpacks
 ---
 
-[TODO #2904](https://github.com/GoogleContainerTools/skaffold/issues/2904)
\ No newline at end of file
+[Buildpacks]((https://buildpacks.io/)) enable building language-based containers from source code, without the need for a Dockerfile.
+
+Skaffold supports building with buildpacks natively.
+Skaffold native buildpacks build process differs from the `pack` command
+`pack build`. 
+
+Skaffold buildpacks support, builds the image inside local docker daemon. 
+It mounts the source dependencies and local artifact cache if caching is enabled 
+to a container in a docker daemon. These get unmounted once the build process is finished.
+
+Once all the necessary data is present, Skaffold will run 
```

Comment:
Maybe we don't need to talk about the lifecycle at all. It's the buildpacks specs, not really something interesting for end users. Or is it?